lms/services/ltia_http.py

```diff
@@ -97,7 +107,6 @@ def _get_new_access_token(self, scopes: list[str]) -> JWTOAuth2Token:
 
 def factory(_context, request):
     return LTIAHTTPService(
-        request.lti_user.application_instance.lti_registration,
```

Comment:
In general I don't like the existing pattern of scoping services to one request, AI or similar.

They become only useful in one context when they could be more useful just passing more parameters to the relevant functions.

Here we are planning to call this from a celery task, for many different registrations.

Reply:
Agreed. Services scoped to the request tend to accidentally become stateful, and later refactors or dependencies between services end up producing accidental crossed data or even memory leaks.

With truly stateless services where the contextual information is passed via method arguments, we could just have a single persistent instance shared by all requests, as you mentioned above.